src/tests/e2etest/CommandRunnerTests.cs

```diff
@@ -47,46 +52,208 @@ public partial class CommandStatus
             public CommandState CurrentState { get; set; }
         }
 
-        [Test]
-        public void CommandRunnerTest_RunCommand_echo_HelloWorld()
+        public partial class ResponseCode
+        {
+            public int ac { get; set; }
+        }
+
+        const int responseStatusSuccess = 200;
+        const int responseStatusFailed = 400;
+
+        const string argumentsPingBing = "ping bing.com";
+        const string argumentsEchoHelloWorld = "echo Hello World";
+        const string helloWorld = "Hello World ";
+        const string argumentsReboot = "reboot";
```

Comment:
> const string argumentsReboot = "reboot"; [](http://example.com/codeflow?start=7&length=41)

This is not used. Looks the reboot test was intended but not completed. Possible to complete it in this PR? Canm be combined with a test to validate command with same id cannot be requested after reboot.

Reply:
This is the same in the 5th iteration...